gradle.properties

```diff
@@ -1,12 +1,22 @@
 projectVersion=3.1.1-SNAPSHOT
-micronautDocsVersion=2.0.0.RC1
-micronautVersion=3.1.0
-micronautTestVersion=2.3.7
+projectGroup=io.micronaut.servlet
+
+micronautDocsVersion=2.0.0
+micronautVersion=3.3.0
+micronautTestVersion=3.0.5
+micronautSecurityVersion=3.3.0
 
 groovyVersion=3.0.9
 spockVersion=2.0-groovy-3.0
-kotlinVersion=1.4.32
-kotlinCoroutinesVersion=1.4.3
+kotlinVersion=1.6.10
+kotlinCoroutinesVersion=1.6.0
+
+googleCloudFunctionsVersion=1.0.4
+tomcatVersion=9.0.58
```

Comment:
native image doesn't work with the latest...  I need to investigate

Reply:
See in `http-server-tomcat/build.gradle` 

```
micronautBuild {
     resolutionStrategy {
         // Because of an issue with GraalVM
         force 'org.apache.tomcat.embed:tomcat-embed-core:9.0.38'
     }
 }
```

Reply:
Yeah, I asked in gitter...  No-one knows why...

After investigation (and trying it out), it seems that it was fixed at some point, and then broken again in 9.0.51 with this commit https://github.com/apache/tomcat/commit/f11dc242ac4f7f6536d10efb876483652304e910

I have selected 9.0.50 for now, as I have no idea currently how to tell graal that class is required

Reply:
Graeme pointed me in the right direction...

It now *seems* to work with the latest

https://github.com/micronaut-projects/micronaut-servlet/pull/271/commits/e7c40538f72da648c299a651ecd9a9112e15cf2a